packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -52,39 +52,21 @@ enum _FloatingActionButtonType {
 /// action button.
 ///
 /// {@tool dartpad}
-/// This example shows how to display a [FloatingActionButton] in a
-/// [Scaffold], with a pink [backgroundColor] and a thumbs up [Icon].
-///
-/// ![](https://flutter.github.io/assets-for-api-docs/assets/material/floating_action_button.png)
+/// This example shows a [FloatingActionButton] in its usual position within a
```

Comment:
Do we need to mention this example is showing the Material 3 FAB?

Reply:
I don't think we'll be updating all the examples to say that when we make M3 the default, so I think it's OK to leave that qualification out here.